frontend/.env.development

```diff
@@ -1,3 +1,4 @@
 REACT_APP_TITLE=LOCAL DEV SimpleReport Prototype
 REACT_APP_BACKEND_URL=http://localhost:8080/graphql
 REACT_APP_OKTA_URL=https://simplereport.cdc.gov/
+REACT_APP_APPINSIGHTS_KEY=27860d2f-45bb-4457-b746-d7174833e292
```

Comment:
Guess it's ok to include this dev key in github? Is the key bound to localhost on the server?

Reply:
it is part of the payload in every request sent by the client to Insights. Given how public it is, I figured it doesn't *really* matter. Also, this is for a dummy instance I spun up while doing dev; we might consolidate with the backend's instance and just kill this one

Reply:
sounds very reasonable.

Reply:
Makes sense to me, I can get you the app insights key for the main logging instance.

I think I can create a custom key that will only allow specific log events to happen.

Reply:
Do we need to have separate keys for different instances (dev, staging, beta, etc.) or is it smart enough to figure it out?

Reply:
I would think we want different ones for prodlike vs other environments, but that's just me spitballing.